app/src/main/java/at/roteskreuz/stopcorona/model/repositories/FilesRepository.kt

```diff
@@ -26,15 +26,16 @@ interface FilesRepository {
     suspend fun createTextFileFromString(text: String, fileName: String)
 
     /**
-     * Write input stream into a new file in the cache storage. If the destination file already
+     * Write input stream into a new file in the app's storage. If the destination file already
      * exists, it will not be overwritten.
      *
      * @param inputStream: The content to write.
      * @param fileName a file name, relative to the apps storage
      *
+     * @return The full path to the file
```

Comment:
Path or file? path would be safer, because we want to know only link.

Reply:
I've already managed it in https://github.com/austrianredcross/stopp-corona-android/pull/138#discussion_r440827965